tests/foreman/ui/test_user.py

```diff
@@ -249,11 +247,25 @@ def test_positive_create_user_different_pass(self, password):
         @Assert: User is created
 
         """
-        name = gen_string('alpha')
+        test_data = valid_strings()
```

Comment:
If we used python3, things would be pretty simple:
```python
test_data = [
    u'foo@!#$^&*( ) {0}'.format(gen_string('alpha', 2)),
    u'bar+{{}}|\"?hi {0}'.format(gen_string('alpha', 2)),
    *valid_strings(),
]
```
However, AFAIK python2 doesn't support this syntax.